Schemas/DF7_10_Roads.hsd.json

```diff
@@ -0,0 +1,1260 @@
+{
+	"schemas": [
+		{
+			"namespace": "http://www.esdi-humboldt.eu/hale/xls",
+			"typeIndex": [
+				{
+					"ref": 0,
+					"name": "ESTATUnitReference"
+				},
+				{
+					"ref": 1,
+					"name": "NAP_MajorRoad"
+				},
+				{
+					"ref": 2,
+					"name": "NAP_MajorRoadCompetentAuthority"
+				},
+				{
+					"ref": 3,
+					"name": "NAP_MajorRoadLimitValues"
+				},
+				{
+					"ref": 4,
+					"name": "NAP_RoadMappingResultDetail"
+				},
+				{
+					"ref": 5,
+					"name": "NAP_RoadReductionHealthImpact_1"
+				},
+				{
+					"ref": 6,
+					"name": "NAP_RoadReductionHealthImpact_2"
+				},
+				{
+					"ref": 7,
+					"name": "NAP_RoadReductionHealthImpact_3"
+				},
+				{
+					"ref": 8,
+					"name": "NAP_RoadReductionMeasure"
+				},
+				{
+					"ref": 9,
+					"name": "NoiseActionPlanMajorRoad"
+				}
```

Comment:
The types for the sheets `DatasetDefaultProperties` and `CodelistProperties` are missing. But it would ok from my perspective to merge it as is and add them in a follow-up PR together with the required mappings.

Reply:
Yes, I already added these types. It will be part of one of my next commits.